core/src/test/java/com/netflix/conductor/core/execution/tasks/TestSubWorkflow.java

```diff
@@ -211,7 +211,7 @@ public void testExecuteWorkflowStatus() {
         subWorkflowInstance.setStatus(Workflow.WorkflowStatus.TIMED_OUT);
         subWorkflowInstance.setReasonForIncompletion("unit2");
         assertTrue(subWorkflow.execute(workflowInstance, task, workflowExecutor));
-        assertEquals(Task.Status.FAILED, task.getStatus());
+        assertEquals(Task.Status.TIMED_OUT, task.getStatus());
```

Comment:
Why was this status changed?

Reply:
Subworkflow task has been updated to closely reflect Workflow status. If the workflow were to `TIMED_OUT`, the task is now set to `TIMED_OUT` as well.